CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Created a bypass for images features on the DC Facelift homepage to not call Repo API. (DR-2873)
```

Comment:
Nitpick, but I would call these image IDs out as swimlanes images in the changelog message so we don't get confused later with the featured images

Reply:
changelog updated 👍